telemetry/definitions/commonDefinitions.json

```diff
@@ -167,6 +167,11 @@
             "type": "int",
             "description": "The amount of time required for the build to complete (in seconds)."
         },
+        {
+            "name": "buildSystemVersion",
+            "type": "string",
+            "description": "The build system version on the user's machine"
```

Comment:
can you make this more descriptive.

Reply:
We're keeping it generic because we support Maven and now Gradle